protected/models/Patient.php

```diff
@@ -190,7 +190,7 @@ public function attributeLabels()
         return array(
             'id' => 'ID',
             'pas_key' => 'PAS Key',
-            'dob' => 'Date of Birth',
+            'dob' => 'Date of Birth <i style="color:grey">(01 Jan 1970)</i>',
```

Comment:
Unfortunately this is a bit of a heavy-handed for just adding a label to the patient creation screen, because it will also affect other screens and potentially some reports as well. It would be cleaner if you just included the label in the patient form.